### ISSUE TITLE
Don't unfollow active/interactive users

### ISSUE THREAD

**Issue by icpgithub** [mifi/instauto#37]
I wonder if it's possible to get a list of accounts that are active/have interacted with you often, and you can have an option to not unfollow them in the "Unfollow auto-followed users" section. i.e. those who have sent you a direct message or replied to a story or liked several posts.

I know there's an "excludeUsers" section you can manually fill it, but if your following grows and you want to auto-unfollow even those who follow you back after a while, but keep certain users who are really interacting with you, it'll be useful if the bot can identify them for you. But I don't know if there's any way to implement that.

**Comment by mifi** [comment]
Yea I don't think it can be done easily

**Comment by icpgithub** [comment]
Yeah, what I manually do (not ideal but easy to do), is to scroll my DM regularly and put the accounts that appear there (either because they've tagged me, or replied to a story or message me) manually in the "excludeusers" section.
